tests/large/test_ide.py

```diff
@@ -450,3 +450,50 @@ def test_default_install(self):
         self.expect_and_no_warn("Arduino is already installed.*\[.*\] ")
         self.child.sendline()
         self.wait_and_close()
+
+
+class BaseNetBeansTests(LargeFrameworkTests):
+    """The base for all Netbeans installers."""
+
+    TIMEOUT_INSTALL_PROGRESS = 600
```

Comment:
Please put back the timeout to 120. The timeout is not the full install time, it's the time when nothing is print/changing on the CLI. So basically: no  download progressing, or anything. If you don't get 0.5-1% of download during those 120 minutes, it means there is a big issue :)